static-site/content/blog/2019-10-21-auspice-v2.md

```diff
@@ -2,6 +2,7 @@
 author: "James Hadfield"
 date: "2019-10-21"
 title: "Auspice v2 released"
+anchorText: "Auspice v2"
```

Comment:
It's necessary to have at least one instance of `anchorText` or else gatsby will complain.